packages/fuchsia_ctl/bin/main.dart

```diff
@@ -186,7 +189,8 @@ Future<OperationResult> ssh(
       identityFilePath: identityFile,
       command: args['command'].split(' '),
       timeoutMs:
-          Duration(milliseconds: int.parse(args['timeout-seconds']) * 1000));
+          Duration(milliseconds: int.parse(args['timeout-seconds']) * 1000),
+      logFilePath: outputFile);
```

Comment:
nit: trailing comma will make this format a bit more nicely

Reply:
done